Second_Seminar/app/src/main/java/org/sopt/androidseminar/ui/SignInActivity.kt

```diff
@@ -36,56 +47,44 @@ class SignInActivity : AppCompatActivity() {
         }
     }
 
-    private fun signUpButtonClickEvent() {
-        binding.registerTV.setOnClickListener {
-            val intent = Intent(this, SignUpActivity::class.java)
-            startActivityForResult(intent, 0)
-        }
+    private fun isIdPwETEmpty() : Boolean{
+        return binding.etGithubId.text.isNullOrBlank() || binding.etGithubPw.text.isNullOrBlank()
```

Comment:
```suggestion
    private fun isIdPwETEmpty() = binding.etGithubId.text.isNullOrBlank() || binding.etGithubPw.text.isNullOrBlank()
```

한줄로 작성할 수 있는 함수는 그냥 "="을 적어도 좋습니당